torch/distributed/_composable/fsdp/_fsdp_init.py

```diff
@@ -0,0 +1,18 @@
+from typing import Union
+
+import torch
+
+
+def _normalize_device(device: Union[torch.device, int, str]) -> torch.device:
```

Comment:
this looks fine, but, why is it an fsdp-specific util, do we not have anything like this in torch already, or maybe we should push it that direction if not?

Reply:
I agree that this is not FSDP-specific. I have not seen a function like this available in Python, but I may have missed it.

I am happy to move this elsewhere if we have an idea of where that should be.

Reply:
there is actually a similar one in torch/distributed/pipeline/sync/worker.py and a non-similar one in test/functorch/test_eager_transforms.py

i would propose to put something in the vicinity of actual torch.Device code to help with this.  cc @ezyang @zou3519 

we can do it in another PR

Reply:
sure. bikeshed a name lol